### PR TITLE
win-error.1.0: remove bad `version` field from metadata

### PR DESCRIPTION
**Pull request by djs55**
See [mirage/ocaml-win-error#6]

Signed-off-by: David Scott <dave@recoil.org>

### CODE REVIEW

File: packages/win-error/win-error.1.0/opam [hunk 1]
```diff
@@ -1,6 +1,5 @@
 opam-version: "2.0"
 name: "win-error"
```
Comment (avsm) [inline packages/win-error/win-error.1.0/opam]:
```suggestion
```

**Comment by avsm** [comment]
did you release it via `dune-release` originally? seems like a bug

**Comment by djs55** [comment]
Hm, I can't remember if I used `dune-release` for this one specifically. I've been trying to use it more consistently but I might not have had it on the laptop I was using.

**Comment by avsm** [comment]
@kit-ty-kate it might be worth putting in an `opam admin check` into check.ocamllabs.io -- I think we still need https://github.com/ocaml/opam/issues/3342 to make the output easier to parse

**Comment by kit-ty-kate** [comment]
> @kit-ty-kate it might be worth putting in an `opam admin check` into check.ocamllabs.io -- I think we still need [ocaml/opam#3342](https://github.com/ocaml/opam/issues/3342) to make the output easier to parse

is `opam admin check` morally the same thing as `opam update --strict`? (at least for this kind of error)

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**